proto/controlclient.proto

```diff
@@ -114,3 +115,20 @@ message _GenerateApiTokenResponse {
   // epoch seconds when the api token expires
   uint64 valid_until = 4;
 }
+
+message _RefreshApiTokenRequest {
+  string api_token = 1;
+  string refresh_token = 2;
+}
```

Comment:
Ergh, I should have thought of this in the design meeting (although this is less a UX design thing and more an implementation detail)... it might be worth doing some testing of how we want the control plane code to look before merging this. I think there's an open question of should the api token come through in a header, like it normally does, and then we do our normal validation of it except it's ok if it's expired, or do we want to have the endpoint be completely unauthenticated and check this as a parameter?

I'm leaning a bit towards it comes through in the header, because that seems easier for the SDK to do (i.e. that's already part of the client configuration)? If it does come through in the header then I don't think we need it here.

Reply:
ah yea thats a good call. Let me play around with it a bit and get back to you here